csharp/extractor/Semmle.Extraction.CSharp.Standalone/BuildAnalysis.cs

```diff
@@ -29,6 +29,9 @@ internal sealed partial class BuildAnalysis : IDisposable
         private readonly Options options;
         private readonly DirectoryInfo sourceDir;
         private readonly DotNet dotnet;
+        private readonly Lazy<IEnumerable<string>> allFiles;
```

Comment:
Why do we need the `Lazy<>`?

Reply:
I guess if neither `UseNuGet` and `ScanNetFrameworkDlls` options are set, then we don't need them.

Reply:
Yes, that is correct.
In any case - I will re-write the code - then this might go away.